test/integration/Ramm/swap.js

```diff
@@ -0,0 +1,154 @@
+const { ethers } = require('hardhat');
+const { expect } = require('chai');
+const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
+
+const setup = require('../setup');
+const { setNextBlockBaseFee, setEtherBalance } = require('../../utils/evm');
+
+const { parseEther } = ethers.utils;
+
+async function getCapitalSupplyAndBalances(p1, tc, tk, memberAddress) {
+  return {
+    ethCapital: await p1.getPoolValueInEth(),
+    nxmSupply: await tc.totalSupply(),
+    ethBalance: await ethers.provider.getBalance(memberAddress),
+    nxmBalance: await tk.balanceOf(memberAddress),
+  };
+}
+
+async function swapSetup() {
+  const fixture = await loadFixture(setup);
+  const { tk, p1, tc } = fixture.contracts;
+  const [member1] = fixture.accounts.members;
+
+  const operator = await tk.operator();
+  await setEtherBalance(operator, parseEther('10000'));
+  await setEtherBalance(member1.address, parseEther('10000'));
+  await setEtherBalance(p1.address, parseEther('145000'));
+
+  await tk.connect(await ethers.getImpersonatedSigner(operator)).mint(member1.address, parseEther('1000000000000'));
```

Comment:
Won't this drop the BV significantly?

Reply:
i didn't thought of that. to be honest, the value came from existing code in integration tests. I have now set it to a more conservative value of `10_000`